sys/suit/suit.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_suit
+ * @{
+ *
+ * @file
+ * @brief       SUIT manifest handler
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @}
+ */
+
+#include "suit.h"
+#include "suit/cbor.h"
+#include "uuid.h"
+#include "luid.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define SUIT_DEVID_BYTES 32
+
+static uuid_t vendor;
```

Comment:
please prepend statics with ```_```. For e.g., "read()" it is totally obvious why, so let's be consistent with the obvious cases. ;)

Reply:
Wouldn't these vars make sense in a context struct?

Reply:
> please prepend statics with `_`. For e.g., "read()" it is totally obvious why, so let's be consistent with the obvious cases. ;)

Ugh, totally right here, not sure why I keep making this mistake

Reply:
> Wouldn't these vars make sense in a context struct?

Maybe, I'm not against grouping them and it might be convenient at some point.

Reply:
> Maybe, I'm not against grouping them and it might be convenient at some point.

If the manifest is part of the image, it would be possible to read it from the other slot. Maybe. ;)

Reply:
Fixed